tests/search/attributesearch/attributesearch.rb

```diff
@@ -56,12 +56,18 @@ def a_r_w_e(word, resnam, explain)
     assert_result("query=fstitle:#{word}", selfdir + "case.#{resnam}.result", nil, nil, 0, explain)
   end
 
-  def test_attributesearch_case
-    deploy_app(SearchApp.new.sd(selfdir+"cased.sd"))
+  # helper method
+  def expect_fail(word, explain)
```

Comment:
The explain parameter is not used.

Reply:
I am keeping it as it is documentation on the call site similar to a_r_w_e usage.
Added a comment why it is unused.